lighthouse-cli/cli-flags.js

```diff
@@ -97,6 +97,8 @@ function getFlags(manualArgv) {
         'max-wait-for-load':
             'The timeout (in milliseconds) to wait before the page is considered done loading and the run should continue. WARNING: Very high values can lead to large traces and instability',
         'extra-headers': 'Set extra HTTP Headers to pass with request',
+        'precomputed-lantern-data-path': 'Path to the file where lantern simulation data should be read from.',
+        'lantern-data-output-path': 'Path to the file where lantern simulation data should be written to.',
```

Comment:
needs more explanation? Unclear from this what "simulation data" is or what it'll be used for